.github/workflows/cli-build-instructions.yml

```diff
@@ -41,15 +41,59 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      # https://github.com/unicode-org/unicodetools/blob/main/docs/uca/index.md#tools--tests
+      # Note: Not running desuffixucd.py in UCA jobs because no version numbers detected in data file names
+      - name: Run command - UCA - collation validity log
+        run: |
+          # invoke main() in class ...UCA.Main
+          MAVEN_OPTS="-ea" mvn -s .github/workflows/mvn-settings.xml exec:java -Dexec.mainClass="org.unicode.text.UCA.Main"  -Dexec.args="writeCollationValidityLog"  -DUVERSION=15.0.0  -pl unicodetools  -DCLDR_DIR=$(cd ../cldr ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd Generated; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)
+          # check for output file
+          compgen -G "Generated/UCA/*/CheckCollationValidity.html"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # TODO: file issue to uncomment, debug errors, and fix?
+      # - name: Run command - UCA - for CLDR & ICU
+      #   run: |
+      #     # invoke main() in class ...UCA.Main
+      #     MAVEN_OPTS="-ea" mvn -s .github/workflows/mvn-settings.xml exec:java -Dexec.mainClass="org.unicode.text.UCA.Main"  -Dexec.args="ICU"  -DUVERSION=15.0.0  -pl unicodetools  -DCLDR_DIR=$(cd ../cldr ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd Generated; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)
+      #   env:
+      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # TODO: file issue to uncomment, debug errors, and fix. Stacktrace observed:
+      # java.lang.AbstractMethodError: Receiver class org.unicode.cldr.util.props.UnicodePropertySymbolTable$ComparisonMatcher does not define or inherit an implementation of the resolved method 'abstract boolean matches(java.lang.Object)' of interface com.ibm.icu.dev.util.CollectionUtilities$ObjectMatcher.
+      #     at org.unicode.cldr.util.props.UnicodeProperty.getSet (UnicodeProperty.java:366)
+      #     at org.unicode.cldr.util.props.UnicodeProperty.getSet (UnicodeProperty.java:327)
+      # ... 
+      #
+      # https://github.com/unicode-org/unicodetools/blob/main/docs/idna.md
+      # - name: Run command - IDNA
+      #   run: |
+      #     # Confirm that the directory for the bin files exists
+      #     compgen -G "Generated/BIN"
+      #     # "Delete all the bin files" as per instructions
+      #     rm -rf Generated/BIN/*
+      #     # run GenerateIdna
+      #     MAVEN_OPTS="-ea" mvn -s .github/workflows/mvn-settings.xml exec:java -Dexec.mainClass="org.unicode.idna.GenerateIdna" -Dexec.args="" -pl unicodetools  -DCLDR_DIR=$(cd ../cldr ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd Generated; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)  -DUVERSION=15.0.0
+      #   env:
+      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
   out-of-source-build:
```

Comment:
optional: My preference is to only test with the out-of-source builds, which are a "stronger" type of test in a way. Duplicate tests/tool runs add to the maintenance burden. It's ok to test/run one thing (or a small number).

Reply:
Done. I updated the name of the CI job for in-source build to say "(Examples)", and did not add any new in-source versions of commands executing the tools listed in the issue.